src/sparkx/Particle.py

```diff
@@ -723,7 +723,7 @@ def pdg_valid(self):
 
         Returns
         -------
-        pdg_valid : int
+        pdg_valid : short
```

Comment:
Maybe a stupid question: why is the return type short instead of bool?

Reply:
I'll change that to bool.